### PR TITLE
make consumable via npm

### PR DESCRIPTION
**Pull request by mdarens**
- add linting

This repo will need to be renamed to knockout.integer after merge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "https://github.com/civicsource/knockout.integer/issues"
+  },
+  "homepage": "https://github.com/civicsource/knockout.integer#readme",
+  "devDependencies": {
+    "babel-eslint": "6.x",
+    "eslint": "2.x",
+    "eslint-plugin-filenames": "1.x"
+  },
+  "peerDependencies": {
+    "knockout": "3.x.0"
```
Comment (chadly) [inline package.json]: Does `3.x.0` work? I think it should just be `3.x`

**Comment by chadly** [comment]
repo renamed

File: package.json (hunk 1 above)
Comment (mdarens) [inline package.json]: me typing: http://i.giphy.com/lJNoBCvQYp7nq.gif

**Comment by chadly** [comment]
:shipit:

**Pull request merged**